src/components/Dashboard.jsx

```diff
@@ -0,0 +1,119 @@
+      {({ user }) => (
+        <div className="App">
+          <div className="card">
+            <div className="card-header">
+              <h1>Welcome, {user.username}</h1>
+            </div>
+            <div className="card-body">
+              <div className="row">
+                <div className="col-4">
+                  <div className="list-group" id="list-tab" role="tablist">
+                    <a
+                      className="list-group-item list-group-item-action active"
+                      id="list-home-list"
+                      data-toggle="list"
+                      href="#list-home"
+                      role="tab"
+                      aria-controls="home"
+                    >
+                      Home
+                    </a>
+                    <a
+                      className="list-group-item list-group-item-action"
+                      id="list-profile-list"
+                      data-toggle="list"
+                      href="#list-profile"
+                      role="tab"
+                      aria-controls="profile"
+                    >
+                      My Profile
+                    </a>
+                    <a
+                      className="list-group-item list-group-item-action"
+                      id="list-messages-list"
+                      data-toggle="list"
+                      href="#list-messages"
+                      role="tab"
+                      aria-controls="messages"
+                    >
+                      My Habits
+                    </a>
+                    <a
+                      className="list-group-item list-group-item-action"
+                      id="list-messages-list"
+                      data-toggle="list"
+                      href="#list-messages"
+                      role="tab"
+                      aria-controls="messages"
+                    >
+                      My Journal
+                    </a>
+                    <a
+                      className="list-group-item list-group-item-action"
+                      id="list-settings-list"
+                      data-toggle="list"
+                      href="#list-settings"
+                      role="tab"
+                      aria-controls="settings"
+                    >
+                      Settings
+                    </a>
+                  </div>
+                </div>
+
+                <div className="col-8">
+                  <div className="tab-content" id="nav-tabContent">
+                    <div className="row">
+                      <div className="col-md-4">
+                        <button type="button" className="btn btn-primary btn-lg">
+                          Current Mood
+                        </button>
+                      </div>
+                      <div className="col-md-4">
+                        <button type="button" className="btn btn-secondary btn-lg">
+                          Ceate New Habit
```

Comment:
`Create`?

Reply:
yup

Reply:
There's `Ceate` instead of `Create` here.

Reply:
oh yes, I caught it.